lib/mirror_matchers.dart

```diff
@@ -2,6 +2,9 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+@deprecated
```

Comment:
Please use `@Deprecated(...)` and include some helpful details!

Reply:
AFAICT no one is ever going to see the message, no one is importing this. I don't really have a concrete migration plan to give since it depends on whether the `name` is known statically or not.

Is it worth the effort?